tools/api_versioning/generate_api_version_header_test.py

```diff
@@ -32,12 +32,12 @@ def tearDown(self):
         pathlib.Path(self._temp_fname).unlink()
 
     # General success pattern when valid file contents is detected.
-    def SuccessfulTestTemplate(self, output_string, current_version: ApiVersion,
-                               oldest_version: ApiVersion):
+    def successful_test_template(self, output_string, current_version: ApiVersion,
```

Comment:
This was missed because flake8 wasn't running properly before?

Reply:
yep